pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -73,6 +73,9 @@ type NodeConfig struct {
 	// DNSConfig controls the DNS configuration.
 	DNSServer *dns.Server
 
+	// TODO: Make this configurable as currently it is not set which means no off-cluster bridging will be performed.
```

Comment:
i have to check, but I think this is a new field and default to off-cluster bridging if not set.